internal/dev_server/adapters/api.go

```diff
@@ -81,21 +82,24 @@ func (a apiClientApi) getFlags(ctx context.Context, projectKey string, href *str
 	})
 }
 
-func (a apiClientApi) getEnvironments(ctx context.Context, projectKey string, href *string) ([]ldapi.Environment, error) {
-	return getPaginatedItems(ctx, projectKey, href, func(ctx context.Context, projectKey string, limit, offset *int64) (*ldapi.Environments, error) {
-		request := a.apiClient.EnvironmentsApi.GetEnvironmentsByProject(ctx, projectKey)
-		if limit != nil {
-			request = request.Limit(*limit)
-		}
```

Comment:
i think we still need this in some form? i don't see the [limit](https://github.com/launchdarkly/ldcli/blob/3dcc22f0280e4b996739476c2bf2282a1ab13bcd/internal/dev_server/ui/src/api.ts#L26) from the frontend being respected

Reply:
oh huh, interesting.  I'll debug this.  Thanks for testing and calling it out!

Reply:
yea, turns out I forgot to add it back 😅